### DISCUSSION TITLE
Tentative release date for milestone 21.1.2

### DISCUSSION THREAD

**Discussion by raghavendran-gopalakrishnan** [keycloak/keycloak#20681] [Q&A] [answered]
Hello folks,

Is there any date planned to release [21.1.2](https://github.com/keycloak/keycloak/milestone/42) ? 

Best regards,
Raghav

**Comment by jonkoops** [comment]
~~Duplicate of #20278. Please do check existing discussions before posting to reduce the amount of notifications maintainers receive.~~ **edit** I was not reading properly, disregard this.

**Reply by raghavendran-gopalakrishnan** [reply]
Thanks for the response, I was interested in the milestone of v21.1.2, under the impression that there might be a interim patch before v22.

**Reply by jonkoops** [reply, marked as answer]
Ah, apologies. I read too fast. Yes, we will be releasing 21.1.2 very soon (in a couple of weeks).

**Reply by raghavendran-gopalakrishnan** [reply]
> ~Duplicate of #20278. Please do check existing discussions before posting to reduce the amount of notifications maintainers receive.~ **edit** I was not reading properly, disregard this.

No problem, cheers.
